SU-SWS/jumpstart_ui
# D8CORE-3493 D8CORE-3513 Wysiwyg video style fixes

# READY FOR REVIEW

# Summary
- Make the wrapper of an iframe that has a caption 100%.
- don't allow floating of a video.

# Need Review By (Date)
- 2/25

# Urgency
- Medium

# Steps to Test
1. checkout this branch & clear browser cache
2. create a basic page and embed a youtube video in a wysiwyg
3. set a caption and save
4. make sure the video isn't tiny
5. remove the caption and set the video to align right or left & save.
6. make sure the video still displays.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
